src/main/scala/com/databricks/spark/avro/avroRelation2/AvroRelation2.scala

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright 2014 Databricks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.databricks.spark.avro.avroRelation2
```

Comment:
I know we did this pattern of `Relation2` for parquet so that we could feature flag the new implementation, but since this is a separate library where the user can pick whatever version they want, I think we should only have one implementation for simplicity.